pages/Useful Utilities/Wallpapers.md

```diff
@@ -1,12 +1,12 @@
 Launch your wallpaper utility with `exec-once=`.
 
-## hyprpaper
+## Hyprpaper
```

Comment:
the names of these projects are NOT capitalized

Reply:
Well they should've been, not my fault y'all can't respect your own projects smh.